### PR TITLE
Adds 4 new snouts

### PR DESCRIPTION
**Pull request by Smol42**
## About The Pull Request

 Very small requested addition of 4 new "acrador" snouts with their specific patterns.

## Proof of Testing

<details>
<summary>↓ ↓ ↓</summary>
  

</details>

## Changelog

:cl: Smol42
add: Added four acrador snouts.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/snout.dm [hunk 1]
```diff
@@ -433,3 +433,27 @@
...
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 3 (Normal)"
+	icon_state = "acrador_3"
+
+/datum/sprite_accessory/snouts/acrador_4
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 4 (Normal)"
+	icon_state = "acrador_4"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/snout.dm:459]: Could you give them names that aren't just 1 to 4, to better represent what they're meant to be?

File: modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: The third one in the proof of testing seems to have the side view having the "top" color of the snout reach down one pixel lower than it should be at the front, meaning that it's not consistent between the front and the side views.

**Review by GoldenAlpharex — DISMISSED** [review]

File: modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi
Comment (Smol42) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: I don't think I see it. 
image

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/snout.dm (hunk 1 above)
Comment (Smol42) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/snout.dm:459]: I've asked a person who requested this addition and they told me that there's no specific names for them, like regional differences, except the pattern sometimes being specific to certain colors.

File: modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi
Comment (GoldenAlpharex) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: Compare it to the front view, you'll see that the gray portion is only two pixels in height from the front.

Comment (vinylspiders) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: Ah I see it, it just needs this pixel here to be lightened to a pink shade to match the front facing view.

image

image

Comment (Smol42) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: > Compare it to the front view, you'll see that the gray portion is only two pixels in height from the front.

pushed a fix: 
image
image
image

Comment (vinylspiders) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: I think there was a misunderstanding--I'll show you exactly what we were talking about. The first front snout was fine, it was just that one pixel on the side view. Shown here are the changes I made locally:

kwOOa7zGI1

3NNKyKiwky

NkOI7IZdd2

If that looks ok to you let me know! I pushed the fix from my local but I can revert it if you're not OK with that.
Just say the word.

Comment (Smol42) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi:1]: > I think there was a misunderstanding--I'll show you exactly what we were talking about. The first front snout was fine, it was just that one pixel on the side view. Shown here are the changes I made locally:
> 
> kwOOa7zGI1
> 
> 3NNKyKiwky
> 
> NkOI7IZdd2
> 
> If that looks ok to you let me know! I pushed the fix from my local but I can revert it if you're not OK with that. Just say the word.

Yeah, it's okay.

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**